api/desecapi/serializers.py

```diff
@@ -779,10 +779,11 @@ def to_internal_value(self, data):
             raise serializers.ValidationError({api_settings.NON_FIELD_ERRORS_KEY: msg})
 
         # add extra fields added by the user
-        unpacked_data.update(**data)
+        data = {**data, **unpacked_data}
```

Comment:
this fixes a problem, add test

Reply:
done, dca593d647678e8e38bc1677f11e4fc97d4405ab